app/models/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
+  # TODO: settings, i.e. user controller
```

Comment:
that goes with the pinging setting, right?

Reply:
highly optional pundit implementation